src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -70,8 +71,9 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       for (Message message : messages) {
         String author = UserStore.getInstance()
           .getUser(message.getAuthorId()).getName();
+        String content = BBCodeParser.parse(message.getContent());
     %>
-      <li><strong><%= author %>:</strong> <%= message.getContent() %></li>
+      <li><strong><%= author %>:</strong> <%= content %></li>
```

Comment:
why not:

<li><strong><%= author %>:</strong> <%= BBCodeParser.parse(message.getContent()) %></li>

I generally only create a local variable if it's used more than once or the line will be too long without it.

Reply:
Sure, I just thought it might improve readability, but I like that rule!